.circleci/config.yml

```diff
@@ -203,7 +203,7 @@ jobs:
     <<: *defaults-release
     steps:
       - better_checkout
-      - run: pyenv global 3.10.2
```

Comment:
Is this change intended?  We're still on 3.10 in addons-server.
(and so on for the other bumps in this file.)

Reply:
Yes, it's because I've updated the ubuntu image to keep up with security updates but Python 3.10 is not available that updated image. This is just to run `dennis` so the mismatch doesn't matter much.